### PR TITLE
Cjlems add bio

### PR DESCRIPTION
**Pull request by cjlems**
@rakshassa No comments from you.

### CODE REVIEW

**Comment by cjlems** [comment]
Closes #24

**Comment by rakshassa** [comment]
But I commented anyway

**Comment by tevtek1** [comment]
denver is great place for hiking...

File: bios/cjlems-bio.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### cjlemsBio
+
+- Lives in Denver, CO
+- Likes Hiking
+- is a noob
```
Comment (rakshassa) [inline bios/cjlems-bio.md:5]: clearly....

File: bios/cjlems-bio.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+### cjlemsBio
+
+- Lives in Denver, CO
```
Comment (rakshassa) [inline bios/cjlems-bio.md]: Actually, it's littleton

File: bios/cjlems-bio.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+### cjlemsBio
+
+- Lives in Littleton, CO
+- Likes Hiking
+- is a noob
+
+#Correct syntax
+@rakshassa Noob.
```
Comment (rakshassa) [inline bios/cjlems-bio.md:8]: hmm, seems incorrect syntax.  You misspelled cjlems as "rakshassa"

**Comment by tevtek1** [comment]
@cjlems looks good :+1: :shipit:

**Pull request merged**